ticketutil/bugzilla.py

```diff
@@ -15,7 +15,7 @@ class BugzillaTicket(ticket.Ticket):
     """
     A BZ Ticket object. Contains BZ-specific methods for working with tickets.
     """
-    def __init__(self, url, project, auth=None, ticket_id=None):
+    def __init__(self, url, project, auth='kerberos', ticket_id=None):
```

Comment:
This should go back to auth=None. BugzillaTicket supports HTTP Basic auth, Kerberos auth, and API Key auth, but we don't want to necessarily set a default of 'kerberos'.

Reply:
I forgot to comment on this. If `auth=None` it breaks on l.67 `elif 'api_key' in self.auth:`. So there is an option to set `auth='kerberos'` as default or somehow deal with auth being None through some additional condition (if not self.auth: logger.error()... elif 'api_key' in self.auth: ... ). I chose the simplest way.

Reply:
Hmm, good point... Ok, I agree. Keep it as `auth='kerberos'`